### PR TITLE
V1.4 UG update

### PR DESCRIPTION
**Pull request by goel-a**
- Update field constraints in prefix and input summary (Note: did not elaborate on email further as there is too much specifics)
- Update sample output message to represent GUI rather than text sample output
- Add constraint to edit command (minimum one attribute must be edited)

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -152,11 +152,11 @@ The table below illustrates the meaning of the prefixes and their respective inp
 | Prefix | Meaning                  | Input       | Constraints |
 |:-------|:-------------------------|:------------|:------------|
 |   -    |                          | INDEX       | Index of applicant specified must not be more than the total number of applicants in the list. |
-| **n/** | Applicant's Name         | NAME        |
-| **p/** | Applicant's Phone Number | PHONE_NUMBER|
-| **e/** | Applicant's Email        | EMAIL       |
-| **j/** | Job Applied              | JOB         |
-| **r/** | Application Round        | ROUND       |
+| **n/** | Applicant's Name         | NAME        | Names can only contain Unicode characters, and cannot contain any numbers. Name's can contain the following special characters: `-` and`'`. **An applicant's name is unique** i.e. two applicants cannot have the same name.
```
Comment (ryantanlien) [inline docs/UserGuide.md:155]: Maybe elaborate on what is Unicode? Or use a different way of expressing it? Most laymen would not be aware of what is Unicode or UTF-8

Comment (goel-a) [reply docs/UserGuide.md]: Changed to a more simple explanation

File: docs/UserGuide.md [hunk 2]
```diff
@@ -152,11 +152,11 @@ The table below illustrates the meaning of the prefixes and their respective inp
...
 |:-------|:-------------------------|:------------|:------------|
 |   -    |                          | INDEX       | Index of applicant specified must not be more than the total number of applicants in the list. |
-| **n/** | Applicant's Name         | NAME        |
-| **p/** | Applicant's Phone Number | PHONE_NUMBER|
-| **e/** | Applicant's Email        | EMAIL       |
-| **j/** | Job Applied              | JOB         |
-| **r/** | Application Round        | ROUND       |
+| **n/** | Applicant's Name         | NAME        | Names can only contain letters (including letters with accents), and cannot contain any numbers. Name's can contain the following special characters: `-` and`'`. **An applicant's name is unique** i.e. two applicants cannot have the same name.
+| **p/** | Applicant's Phone Number | PHONE_NUMBER| Phone numbers must only contain numbers, and must be between 3 and 15 characters long.
+| **e/** | Applicant's Email        | EMAIL       | Emails should be of the format `local-part@domain`.
```
Comment (zzhikai) [inline docs/UserGuide.md:157]: could you add in, like 2 examples of domain, 'domain can be mail and mail.com' just a short line

**Review by zzhikai — APPROVED** [review]
LGTM, do consider the comment i left though its not a big issue

File: docs/UserGuide.md (hunk 2 above)
Comment (goel-a) [reply docs/UserGuide.md]: Added 2 examples each of a valid and invalid email

**Pull request merged**